test/trust_is_risk.js

```diff
@@ -52,15 +54,26 @@ describe("TrustIsRisk", () => {
 
     trustDecreasingMTX = new MTX({
       inputs: [
-        inputOneOfTwoMultisig
+        inputOneOfThreeMultisig
       ],
       outputs: [
-        testHelpers.getOneOfTwoMultisigOutput(addr.alice.pubKey, addr.bob.pubKey, 20 * COIN),
+        testHelpers.getOneOfThreeMultisigOutput(addr.alice.pubKey, addr.bob.pubKey, 20 * COIN),
         testHelpers.getP2PKHOutput(addr.alice.base58, 22 * COIN)
       ]
     });
   });
 
+  describe("tag", () => {
+    it("corresponds to a valid public key", () => {
+      Buffer.isBuffer(tir.fakePubKey).should.equal(true);
+      secp256k1.publicKeyVerify(tir.fakePubKey).should.equal(true);
+    });
+
+    it("is a valid bitcoin address", () => {
+      assert(bcoin.primitives.Address.fromBase58(tir.tag.toString("ascii")));
```

Comment:
`!!(bcoin.primitives.Address.fromBase58(tir.tag.toString("ascii"))).should.equal(true)`

Reply:
Done